src/extensions/default/CodeFolding/Prefs.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * Wrapper around brackets pref system to ensure preferences are stored in in one single object instead of using multiple keys.
+ * This is to make it easy for the user who edits their preferences file to easily manage the potentially numerous lines of preferences generated by the persisting code-folding state.
+ * @author Patrick Oladimeji
+ * @date 3/22/14 20:39:53 PM
+ */
+/*jslint vars: true, plusplus: true, devel: true, nomen: true, indent: 4, maxerr: 50 */
+/*global define, brackets*/
+define(function (require, exports, module) {
+    "use strict";
+    var PreferencesManager      = brackets.getModule("preferences/PreferencesManager"),
+        prefs                   = PreferencesManager.getExtensionPrefs("code-folding"),
+        strings                 = brackets.getModule("strings"),
+        store                   = {},
+        foldsKey                = "folds";
+
+    //default preference values
+    prefs.definePreference("enabled", "boolean", true,
+                           {name: strings.ENABLE_CODE_FOLDING, description: strings.ENABLE_CODE_FOLDING});
+    prefs.definePreference("minFoldSize", "number", 2,
+                           {name: strings.MIN_FOLD_SIZE, description: strings.MIN_FOLD_SIZE_HELP});
+    prefs.definePreference("saveFoldStates", "boolean", true,
+                           {name: strings.SAVE_FOLD_STATES, description: strings.SAVE_FOLD_STATES_HELP});
+    prefs.definePreference("alwaysUseIndentFold", "boolean", true,
+                           {name: strings.ALWAYS_USE_INDENT_FOLD, description: strings.ALWAYS_USE_INDENT_FOLD_HELP});
```

Comment:
It means that line indentation is used as a suggestion for fold regions. Although this shouldn't take priority over range finders based on the current mode. So where there is conflict, the brace-fold-finder should take precedence. Having said that this might be another pref that could be better off disabled by default.